src/server/database/models/index.js

```diff
@@ -28,9 +28,9 @@ dates.belongsTo(workers, {
 workers.hasMany(dates, { foreignKey: 'worker_name' });
 
 participantCourses.belongsTo(courses, {
-  onDelete: 'CASCADE', foreignKey: 'course_id', targetKey: 'id',
+  onDelete: 'CASCADE', foreignKey: 'course_name', targetKey: 'course_name',
```

Comment:
@AsalaKM I think this is the line I intended, with my previous comment 😀Sorry about all the confusion!

Here, since `targetKey` is using course_name, there's the danger I described previously, where a course could have its name updated by an admin and then existing participantCourses which referred to the old name would be broken.  Using `targetKey: id` should keep all of the associations safe when courses are updated with new names.

I think the same is true for the dates.belongsTo() assocation above, which is using `targetKey: 'username'` - if a worker name changed, then the dates -> workers association would point to old usernames (and would be broken).

@mohannadhanafi FYI ^^ since you are currently working on the date / workers add functionality.

Reply:
:ok_hand: